### PR TITLE
 chore: add bash_profile fallback for bash

### LINKED ISSUE #1553: Install script doesn't properly add to $PATH on macOS with Bash
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: irrelevant
- Spicetify version: 2.9.3
```

### 📝 Description

If running macOS with Bash and not Zsh, the script adds spicetify to the $PATH for `.bashrc`. MacOS runs the `.bash_profile` and not `.bashrc`. Someone on Discord had this problem and we fixed it by adding a symlink for `.bash_profile` to `.bashrc`. 

### 📸 Screenshots

_No response_

### PR DESCRIPTION
**Pull request by afonsojramos**
Closes #1553.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -80,7 +80,13 @@ check() {
 
 case $SHELL in
 	*zsh) check ".zshrc" ;;
-	*bash) check ".bashrc" ;;
+	*bash) 
+		if [ -f "$HOME/.bashrc" ];
```
Comment (wael444) [inline install.sh:84]:
```suggestion
		if [ -f "$HOME/.bashrc" ];
```

File: install.sh [hunk 2]
```diff
@@ -80,7 +80,13 @@ check() {
...
+		then
+			check ".bashrc"
+		else
+			check ".bash_profile"
+		fi;;
```
Comment (wael444) [inline install.sh:89]:
```suggestion
	*bash) 
		[ -f "$HOME/.bashrc" ] && check $HOME/.bashrc
		[ -f "$HOME/.bash_profile" ] && check $HOME/.bash_profile
	;;
```

Comment (afonsojramos) [reply install.sh]: Blindly accepted the change on GitHub, which broke functionality, but already fixed in [591d0ec](https://github.com/spicetify/spicetify-cli/pull/1583/commits/591d0ece9493b79fa9f342c14c73f2c6bd82a07c).

Comment (wael444) [reply install.sh]: oh crap, i forgot that `check` actually already does the `$HOME` path...

**Pull request merged**